src/myspace/PersonalDetails.jsx

```diff
@@ -0,0 +1,293 @@
+import React, { useState } from "react";
+import { BASE_API } from "../config";
+import Cookies from "js-cookie";
+import "./MySpace.css";
+
+
+export default function PersonalDetails() {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [personalDetails, setPersonalDetails] = useState({});
+  // const [username, setUsername] = useState(null);
+  // const [email, setEmail] = useState(null);
+  // const [name, setName] = useState(null);
+  // const [ircId, setIrcId] = useState(null);
+  // const [mentor, setMentor] = useState(null);
+  // const [mentee, setMentee] = useState(null);
```

Comment:
Please even while trying no comments. If something doesn't work it's never lost. You can always get it from the previous commit

Reply:
got it. fixed now.